content/source/docs/enterprise/before-installing/network-requirements.html.md

```diff
@@ -56,7 +56,7 @@ When [Cost Estimation](/docs/enterprise/admin/integration.html#cost-estimation-i
 
 * `api.pricing.us-east-1.amazonaws.com`
 * `cloud.google.com`
-* `azure.microsoft.com`
+* `ratecard.azure-api.net`
```

Comment:
```suggestion
* `management.azure.com`
* `ratecard.azure-api.net`
```

@mwhooker helped dig into this, and sure enough: we call out to the management.azure hostname, and it 302s to the one you found. Thank you again for finding this! I'll just add the second hostname in there and ship it.